client/src/utils/css-help.ts

```diff
@@ -86,6 +86,7 @@ class CSSHelp {
     const link: HTMLLinkElement | null = this.doc?.querySelector(
       "link[href*='styles']"
     );
+    // Most* browser extensions inject styles with class/media attributes
     const style: HTMLStyleElement | null = this.doc?.querySelector(
       'style:not([class][media])'
     );
```

Comment:
Not sure what happened to my code selection, I'm talking about this selector

Reply:
It must not have either.

Reply:
Maybe I can make it more clear.

As it is now, your selector will *not* return the style element only if the element has both attributes. If either of the attributes is present it will return the style element.

Reply:
@lasjorg I have been trying to figure this out, but I am not too sure...

Reply:
I think you got it :+1:

Reply:
Just as information, in Selectors Level 4 `:not()` accepts a list of selectors. I'm just not sure how great the support is. As far as I can tell it should be supported in Chrome, Firefox, and WebKit

> https://caniuse.com/?search=%3Anot()
> 
> Selectors Level 4 allows `:not()` to accept a list of selectors. Thus, `:not(a):not(.b):not([c])` can instead be written as `:not(a, .b, [c])` and `:not(a.b[c])` works as intended.

So this should work as well:

`document.querySelector('style:not([class], [media])');`

But I think your selector is safer. From the testing I did it works as intended.